Sources/Lumina/Camera/Extensions/CameraActionsExtension.swift

```diff
@@ -19,14 +19,11 @@ extension LuminaCamera {
   func captureStillImage() {
     LuminaLogger.info(message: "Attempting photo capture")
     var settings = AVCapturePhotoSettings()
-    if #available(iOS 11.0, *) {
-      if self.photoOutput.availablePhotoCodecTypes.contains(.hevc) {
-        LuminaLogger.notice(message: "Will capture photo with HEVC codec")
-        settings = AVCapturePhotoSettings(format: [AVVideoCodecKey: AVVideoCodecType.hevc])
-      }
+    if self.photoOutput.availablePhotoCodecTypes.contains(.hevc) {
+      LuminaLogger.notice(message: "Will capture photo with HEVC codec")
+      settings = AVCapturePhotoSettings(format: [AVVideoCodecKey: AVVideoCodecType.hevc])
     }
     switch self.torchState {
-        //swiftlint:disable empty_enum_arguments
       case .on(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/45e47f954408babfc22eace2d454b922cb0007fe/Sources/Lumina/Camera/Extensions/CameraActionsExtension.swift#L27"></span>Arguments can be omitted when matching enums with associated values if they are not used.<br />
<code>empty_enum_arguments</code> <code>CameraActionsExtension.swift:27</code></td>
    </tr>
  </tbody>
</table>